### DISCUSSION TITLE
Yet another engineer

### DISCUSSION THREAD

**Discussion by v01d-4rm463dd0n** [WeAreMahsaAmini/Core#424] [Introductions]
This is v01d, fullstack/frontend engineer. Bring it on!

**Comment by Pink3lephant** [comment]
Hi @v01d-4rm463dd0n 

Nice to hear from you. We are working on a project to distribute knowledge on how to create VPN servers that actually work (think of it as meta-Outline, where you can setup several protocols). 

At the moment we are working on our landing page (Nextjs / Strapi], it's in progress but we need more resources to finish it faster. 

Can you please have a look and let me know if you're interested in helping out? 

🐜  Current website: [Antyfilter.com](https://Antyfilter.com)
👨‍💻 [Github](https://github.com/Antyfilter)


You can email me at `freeinternettech` [at] gmail. 

To freedom 🕊️

**Comment by tcf-admin** [comment]
Hi @v01d-4rm463dd0n, 
check out forDem's introduction here: https://github.com/WeAreMahsaAmini/Core/discussions/495
Hope you find it interesting and we can have you next to us.
